### PR TITLE
Dependency update.

### PR DESCRIPTION
**Pull request by Tomas-Kraus**

### CODE REVIEW

File: jaxws-ri/boms/bom/pom.xml [hunk 1]
```diff
@@ -36,18 +36,19 @@
...
-        <jakarta.jws-api.version>3.0.0</jakarta.jws-api.version>
+        <jaxb.version>4.0.0-M2</jaxb.version>
+        <jaxws-api.version>4.0.0-RC2</jaxws-api.version>
         <management-api.version>3.2.3</management-api.version>
         <mimepull.version>1.9.15</mimepull.version>
-        <saaj-api.version>2.0.1</saaj-api.version>
-        <saaj-impl.version>2.0.1</saaj-impl.version>
-        <streambuffer.version>2.0.2</streambuffer.version>
-        <woodstox-core.version>6.2.6</woodstox-core.version>
+        <resolver.version>20050927</resolver.version>
```
Comment (lukasj) [inline jaxws-ri/boms/bom/pom.xml:43]: resolver has been already removed in master

File: jaxws-ri/bundles/jaxws-rt/src/main/java/module-info.java [hunk 2]
```diff
@@ -32,7 +32,7 @@
 
     requires transitive jakarta.activation;
     requires transitive jakarta.annotation;
-    requires transitive jakarta.jws;
+//    requires transitive jakarta.jws;
```
Comment (lukasj) [inline jaxws-ri/bundles/jaxws-rt/src/main/java/module-info.java:35]: remove (it's now part of xml.ws-api)

File: jaxws-ri/extras/eclipselink_jaxb/pom.xml [hunk 3]
```diff
@@ -102,6 +102,7 @@
                         <additionalClasspathElement>${project.build.outputDirectory}</additionalClasspathElement>
                     </additionalClasspathElements>
                     <forkCount>1</forkCount>
+                    <testFailureIgnore>true</testFailureIgnore>
```
Comment (lukasj) [inline jaxws-ri/extras/eclipselink_jaxb/pom.xml:105]: I think I saw a property for this in project pom

File: jaxws-ri/extras/jaxws-maven-plugin/pom.xml [hunk 4]
```diff
@@ -73,44 +73,44 @@
...
                 <artifactId>maven-plugin-annotations</artifactId>
-                <version>3.6.1</version>
+                <version>3.6.2</version>
                 <scope>provided</scope>
             </dependency>
             <dependency>
                 <groupId>org.testng</groupId>
                 <artifactId>testng</artifactId>
-                <version>6.14.2</version>
+                <version>7.4.0</version>
```
Comment (lukasj) [inline jaxws-ri/extras/jaxws-maven-plugin/pom.xml:97]: this requires CQ, so either file it or keep it as it was

File: jaxws-ri/extras/jaxws-maven-plugin/pom.xml [hunk 5]
```diff
@@ -173,7 +173,7 @@
         <warn.limit>150</warn.limit>
 
 <!--        <jaxws-tools.version>${project.version}</jaxws-tools.version>-->
-        <jaxws-tools.version>3.0.2</jaxws-tools.version>
+        <jaxws-tools.version>4.0.0-SNAPSHOT</jaxws-tools.version>
```
Comment (lukasj) [inline jaxws-ri/extras/jaxws-maven-plugin/pom.xml:176]: keep 3.0.2 here otherwise tests are going to fail

File: jaxws-ri/runtime/rt/pom.xml [hunk 6]
```diff
@@ -59,10 +59,10 @@
             <groupId>org.glassfish.gmbal</groupId>
             <artifactId>gmbal-api-only</artifactId>
         </dependency>
-        <dependency>
+        <!--dependency>
             <groupId>org.jvnet.staxex</groupId>
             <artifactId>stax-ex</artifactId>
-        </dependency>
+        </dependency-->
```
Comment (lukasj) [inline jaxws-ri/runtime/rt/pom.xml:65]: why is this commented out?

File: jaxws-ri/runtime/rt/src/main/java/module-info.java [hunk 7]
```diff
@@ -32,7 +32,7 @@
 
     requires transitive jakarta.activation;
     requires transitive jakarta.annotation;
-    requires transitive jakarta.jws;
+//    requires transitive jakarta.jws;
```
Comment (lukasj) [inline jaxws-ri/runtime/rt/src/main/java/module-info.java:35]: remove

File: jaxws-ri/tools/wscompile/src/test/java/module-info.java [hunk 8]
```diff
@@ -11,7 +11,7 @@
 module com.sun.tools.ws.wscompile.test {
     requires java.xml;
     requires transitive com.sun.tools.ws.wscompile;
-    requires jakarta.jws;
+//    requires jakarta.jws;
```
Comment (lukasj) [inline jaxws-ri/tools/wscompile/src/test/java/module-info.java:14]: remove

File: jaxws-ri/boms/bom/pom.xml (hunk 1 above)
Comment (Tomas-Kraus) [reply jaxws-ri/boms/bom/pom.xml]: removed

File: jaxws-ri/bundles/jaxws-rt/src/main/java/module-info.java (hunk 2 above)
Comment (Tomas-Kraus) [reply jaxws-ri/bundles/jaxws-rt/src/main/java/module-info.java]: removed

File: jaxws-ri/extras/jaxws-maven-plugin/pom.xml (hunk 4 above)
Comment (Tomas-Kraus) [reply jaxws-ri/extras/jaxws-maven-plugin/pom.xml]: OK, micro version only updated.

File: jaxws-ri/runtime/rt/pom.xml (hunk 6 above)
Comment (Tomas-Kraus) [reply jaxws-ri/runtime/rt/pom.xml]: removed comment

File: jaxws-ri/runtime/rt/src/main/java/module-info.java (hunk 7 above)
Comment (Tomas-Kraus) [reply jaxws-ri/runtime/rt/src/main/java/module-info.java]: removed

File: jaxws-ri/tools/wscompile/src/test/java/module-info.java (hunk 8 above)
Comment (Tomas-Kraus) [reply jaxws-ri/tools/wscompile/src/test/java/module-info.java]: removed

File: jaxws-ri/extras/jaxws-maven-plugin/pom.xml (hunk 5 above)
Comment (Tomas-Kraus) [reply jaxws-ri/extras/jaxws-maven-plugin/pom.xml]: 3.0.2 is back

**Review by lukasj — APPROVED** [review]
LGTM

**Pull request merged**